scripts/gulpfiles/release_tasks.js

```diff
@@ -160,30 +170,25 @@ const publish = gulp.series(
 // Publish a beta version of Blockly.
 const publishBeta = gulp.series(
   updateBetaVersion,
-  buildTasks.cleanBuildDir,
-  buildTasks.build,
-  buildTasks.checkinBuilt,
+  recompile,
   packageTasks.package,
   checkBranch,
   checkReleaseDir,
   loginAndPublishBeta
 );
 
-// Switch to a new branch, update the version number, and build Blockly.
-const recompile = gulp.series(
+// Switch to a new branch, update the version number, build Blockly
+// and check in the resulting built files.
+const recompileDevelop = gulp.series(
   gitTasks.syncDevelop(),
   gitTasks.createRebuildBranch,
   updateVersionPrompt,
-  buildTasks.cleanBuildDir,
-  buildTasks.build,
-  buildTasks.checkinBuilt,
-  typings.typings,
-  typings.checkinTypings,
+  recompile,
   gitTasks.pushRebuildBranch
   );
 
 module.exports = {
-  recompile: recompile,
+  recompile: recompileDevelop,
```

Comment:
Nit: now we have an exported command named `recompile` and an internal function named `recompile` and they don't match. Please change the internal one to `recompileInternal` to avoid confusion down the line.

Reply:
Good point.  I opted for `rebuildAll` instead of `recompileInternal`, because it is a little more clear about the scope of what is done.